kernel/kernel-api/src/main/java/io/delta/kernel/internal/TableImpl.java

```diff
@@ -44,18 +44,14 @@ public static Table forPath(TableClient tableClient, String path)
         return new TableImpl(logPath, dataPath);
     }
 
-    private final Path logPath;
```

Comment:
Now that we are doing this refactor ... we should also create a `Table::getPath` API. It's silly that we can create a table instance that represents a delta table at a particular path, but cannot ask it what the path is.

I don't _think_ we need a `getLogPath` API yet?

Reply:
Added getPath API